src/API/src/db/bionomics/entities/bionomics.entity.ts

```diff
@@ -110,6 +111,13 @@ export class Bionomics extends BaseEntity {
 
   // Associations
 
+  @ManyToOne(() => Dataset, (dataset) => dataset.bionomics, {
+    eager: true,
+    cascade: true,
```

Comment:
We probably want `cascade: ['insert, update']` here. `cascade: true` tells the db to delete this child entity when the parent is deleted, so it will try and delete the dataset entity when the bionomics entity is deleted. But we want the dataset entities to stick around.